tests/test_typed_event_functions.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 Google LLC
```

Comment:
I think we should add a conformance test too, though we can do this in a follow up PR if you prefer


It's going to need a funciton here https://github.com/GoogleCloudPlatform/functions-framework-python/blob/master/tests/conformance/main.py that returns "OK", 200  (is that going to work, is that a builtin?) that the conformance test expects for HTTP functions



The `from_dict` for whatever class you add will need to be able to handle any HTTP JSON payload, so probably convert to like an actual dictionary in python? or maybe a string or something if that makes sense

We can treat it like `functionType` http here: https://github.com/GoogleCloudPlatform/functions-framework-python/blob/master/.github/workflows/conformance.yml

Reply:
"OK", 200 will work as it is a builtin. 
 "The from_dict for whatever class you add will need to be able to handle any HTTP JSON payload, so probably convert to like an actual dictionary in python? or maybe a string or something if that makes sense" I am sorry but I did not understand this.

Reply:
I agree that we should add a conformance test. I can add it in a followup PR